scripts/hudson/narayana.sh

```diff
@@ -56,6 +56,7 @@ function init_test_options {
     [ x"$CODE_COVERAGE_ARGS" != "x" ] || CODE_COVERAGE_ARGS=""
     [ $ARQ_PROF ] || ARQ_PROF=arq	# IPv4 arquillian profile
     [ $IBM_ORB ] || IBM_ORB=0
+    [ $ENABLE_TRACE_LOGS ] || ENABLE_TRACE_LOGS=" -Ptrace.thorntail.test -Ptrace.lra.coordinator"
```

Comment:
Maybe this variable should be named more precisely? What I mean is that this `ENABLE_TRACE_LOGS` is meant to work really only for the `rts/lra` maven module. It's not beneficial for any other Narayana maven module. Could be rather something like `ENABLE_LRA_TRACE_LOGS`?

Reply:
Okay, I will the changes.